forms.lua

```diff
@@ -1,173 +1,119 @@
+			"item_image_button[8.5,3;1,1;" .. "currency:minegeld_10" .. ";c-1000;\n\n\b\b\b\b\b" .. "100" .. "]" ..
+			"item_image_button[9.5,3;1,1;" .. "currency:minegeld_50" .. ";c-5000;\n\n\b\b\b\b\b" .. "100" .. "]" ..
+			"item_image_button[10.5,3;1,1;" .. "currency:minegeld_100" .. ";c-10000;\n\n\b\b\b\b\b" .. "100" .. "]"
+	end
+	formspec = formspec ..
+		"list[current_player;main;2,4.25;8,1;]" ..
+		"list[current_player;main;2,5.5;8,3;8]" ..
+		"listring[]" .. default.get_hotbar_bg(2, 4.25)
 
-function atm.showform3 (player)
-	atm.read_account(player:get_player_name())
-	local formspec =
-	"size[8,8.5]"..
-	default.gui_bg..
-	default.gui_bg_img..
-	default.gui_slots..
-	"label[1.25,0.5;Money input]" ..
-	"label[5.25,0.5;Money output]" ..
-	"label[2.5,0.15;Your account balance: ".. atm.balance[player:get_player_name()].. " " .. MONEY_SYMBOL .. "]" ..
-	"button_exit[3.5,2.75;1,2;Quit;Quit]" ..
-	"item_image_button[0.5,1;1,1;".. "currency:minegeld" ..";i1;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[1.5,1;1,1;".. "currency:minegeld_5" ..";i5;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[2.5,1;1,1;".. "currency:minegeld_10" ..";i10;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[4.5,1;1,1;".. "currency:minegeld" ..";i-1;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[5.5,1;1,1;".. "currency:minegeld_5" ..";i-5;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[6.5,1;1,1;".. "currency:minegeld_10" ..";i-10;\n\n\b\b\b\b\b" .. "1" .."]" ..
-	"item_image_button[0.5,2;1,1;".. "currency:minegeld" ..";t10;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[1.5,2;1,1;".. "currency:minegeld_5" ..";t50;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[2.5,2;1,1;".. "currency:minegeld_10" ..";t100;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[4.5,2;1,1;".. "currency:minegeld" ..";t-10;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[5.5,2;1,1;".. "currency:minegeld_5" ..";t-50;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[6.5,2;1,1;".. "currency:minegeld_10" ..";t-100;\n\n\b\b\b\b\b" .. "10" .."]" ..
-	"item_image_button[0.5,3;1,1;".. "currency:minegeld" ..";c100;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[1.5,3;1,1;".. "currency:minegeld_5" ..";c500;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[2.5,3;1,1;".. "currency:minegeld_10" ..";c1000;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[4.5,3;1,1;".. "currency:minegeld" ..";c-100;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[5.5,3;1,1;".. "currency:minegeld_5" ..";c-500;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"item_image_button[6.5,3;1,1;".. "currency:minegeld_10" ..";c-1000;\n\n\b\b\b\b\b" .. "100" .."]" ..
-	"list[current_player;main;0,4.25;8,1;]"..
-	"list[current_player;main;0,5.5;8,3;8]"..
-	"listring[]"..
-	default.get_hotbar_bg(0, 4.25)
-	minetest.after((0.1), function(gui)
-			return minetest.show_formspec(player:get_player_name(), "atm.form3", gui)
-		end, formspec)
```

Comment:
perhaps this escape mess could be more neatly done? store in a variable, function wrap, etc `\n\n\b\b\b\b\b`